### PR TITLE
chore: update kr translation json

### PR DESCRIPTION
**Pull request by tabathadelane**

### CODE REVIEW

File: src/i18n/translations/kr/translation.json [hunk 1]
```diff
@@ -1,64 +1,73 @@
 {
-  "404": {
-    "statusCode": "404",
-    "errorMessage": "잘못된 URL이거나, 존재하지 않는 페이지입니다. <2>홈페이지로 돌아가기</2>"
+  "404" : {
+    "statusCode" : "404",
+    "errorMessage" : "잘못된 URL이거나, 존재하지 않는 페이지입니다. &lt;2&gt;홈페이지로 돌아가기&lt;/2&gt;"
```
Comment (clarkmcadoo) [inline src/i18n/translations/kr/translation.json:4]: I think this can just be made `<2>` since its in a string, but i'm really not sure what this is...

Comment (tabathadelane) [reply src/i18n/translations/kr/translation.json]: i totally spaced this bug being a possibility here. pushed up a fix

**Review by roadlittledawn — APPROVED** [review]
looks gute! 

noticed this page had two feedback forms for some reason though ¯\_(ツ)_/¯

¯\_(ツ)_/¯

**Comment by tabathadelane** [comment]
> * Deploy preview
https://docswebsitedevelop-tabathaupdatetranslationkr.gatsbyjs.io/kr/docs/apm/agents/php-agent/installation/php-agent-installation-overview/

its also in production
https://docs.newrelic.com/kr/docs/apm/agents/php-agent/installation/php-agent-installation-overview/

 ¯_(ツ)_/¯

**Comment by tabathadelane** [comment]
InstallFeedback dupe fixed [here](https://github.com/newrelic/docs-website/pull/10728#pullrequestreview-1218003606)

**Pull request merged**